Framework/Algorithms/src/SumSpectra.cpp

```diff
@@ -379,7 +413,7 @@ void SumSpectra::doRebinnedOutput(MatrixWorkspace_sptr outputWorkspace,
   // Loop over spectra
   for (const auto i : m_indices) {
     // Don't go outside the range.
-    if ((i >= m_numberOfSpectra) || (i < 0)) {
+    if ((i >= this->m_numberOfSpectra) || (i < 0)) {
```

Comment:
You're right, there really is not much point to using `this->` - while my IDE does not highlight it, the `m_` naming convention for member variables implies the same anyway. I have removed it throughout the file in the interest of consistency